hw/ip/clkmgr/doc/_index.md

```diff
@@ -0,0 +1,191 @@
+Each group has specific attributes and controls whether software is allowed to influence individual clocks during the active power state.
+For low power states, please see [power manager]({{< relref "hw/ip/pwrmgr/doc" >}}).
+
+The grouping is derived from the chip partition and related security properties.
+For illustrative purposes, this document uses the following assumed chip partition
+
+![Example chip partition](example_chip_partition.svg)
+
+The actual partition may differ per design, however the general principles are assumed to be the same.
+Each group can be made up of more than 1 source clock.
+The clocks themselves may be asynchronous - the grouping is thus a logical grouping instead of a physical one.
+
+The grouping is summarized in the table below and described in more detail afterwards.
+The table shows the group name, the modules that belong to each group, and whether SW can directly (via register control) or indirectly (via wait-for-interrupt) influence the state of the clock in the form of clock gating.
+
+| Group           | Frequencies                    | Modules                                                        | Software       | Wait for Interrupt |
+| -------------   | ------------------------------ | -------------------------------------------------------------- | -------------- | ------------------ |
+| Power-up        | 100~200KHz, 24MHz              | Clock Manager, Power Manager, Reset Manager                    | No             | No                 |
+| Transactional   | ~100MHz                        | Aes, Hmac, Key Manager, Otbn                                   | Yes (1)        | Yes (2)            |
+| Infrastructural | 24MHz, ~100MHz                 | Fabric, Fabric gaskets (iopmp), Memories                       | No             | Yes (3)            |
+| Security        | 24MHz, ~100MHz                 | Alert handler, Entropy, Life cycle, Pinmux, Plic, Sensors      | No             | No                 |
+| Peripheral      | 24MHz, 48MHz, 96MHz            | I2c, Spi, Uart, Usb, others                                    | Yes            | Yes                |
+| Timers          | 100-200KHz, 24MHz              | AON timers, Timers, Watchdog                                   | No             | No                 |
+
+* 1 - Transactional clock group's software control is only a software hint.
+* 2 - Transactional clock group's wait-for-interrupt control is only a hint.
+* 3 - May require additional complexity to handle multi-host (non-wait-for-interrupt) scenarios
+
+## Power-up Clock Group
+
+The group refers to modules responsible for power up, such as power, reset and clock managers.
+Large portions of these modules operate to release clocks and resets for the rest of the design, thus cannot operate on gated versions of the clocks themselves.
+They are the only group running clocks directly from the source.
+All follow groups are derived after root clock gating.
+See [block diagram]() for more details
```

Comment:
I think this inline link/image is incomplete?

Reply:
ah thank you. completely forgot to update this.